app/views/activities/home.html.erb

```diff
@@ -0,0 +1,59 @@
+<div class="row">
+	<div class="col-md-4 p-5 mx-auto text-center">
+
+		<h1 class="title">Dashboard</h1>
+		
+		<% if current_user.image_name? %>
+			<%= image_tag current_user.image_name.url, class: "dashboard_image" %>
+		<% else %>
+			<%= image_tag "user_images/default_user.png", class: "dashboard_image" %>
+		<% end %>
+
+		<div class="user_name mt-2">
+			<%= current_user.name %>
+		</div>
+	
+		<div class="mt-3">
+			<%= link_to "learned #{@user.words.count} words", activity_path(@user), class: "dashboard_list" %>
+		</div>
+		<div class="mt-3">
+			<%= link_to "learned #{@activity.count} lessons", categories_path, class: "dashboard_list" %>
+		</div>
+
+		
+	</div>
+
+	<div class="col-md-8 p-5">
+		<h1 class="title">Activities</h1>
+
+			<div class="activity">
+				<% @activities.each do |activity| %>
```

Comment:
@saki4567 This loop of code is also used in other files. Please Use partial file to avoid redundancy.
Try to implement rendering a collection, try to refer here:
https://guides.rubyonrails.org/layouts_and_rendering.html#rendering-collections